frontend/src/metabase/admin/permissions/pages/DataPermissionsPage/DataPermissionsPage.tsx

```diff
@@ -48,10 +53,23 @@ function DataPermissionsPage({
 
   const loadPermissions = () => dispatch(loadDataPermissions());
```

Comment:
This probably needs to be updated so that we re-load from state, rather than make a new API call